CMakeLists.txt

```diff
@@ -469,7 +469,7 @@ set (CMAKE_POSTFIX_VARIABLE "CMAKE_${CMAKE_BUILD_TYPE_UC}_POSTFIX")
 
 if (USE_STATIC_LIBRARIES)
     set (CMAKE_POSITION_INDEPENDENT_CODE OFF)
-    if (OS_LINUX AND NOT ARCH_AARCH64)
+    if (OS_LINUX AND NOT ARCH_AARCH64 AND NOT ARCH_S390X)
```

Comment:
The comment in l. 473/4 would need an update. But perhaps position-independent builds do work on s390x.